docs/docs/04-standard-library/02-std/api-reference.md

```diff
@@ -2679,6 +2688,64 @@ Converts a Json to a Struct, returning nil if the Json is not valid.
 
 
 
+### StructSchema <a name="StructSchema" id="@winglang/sdk.std.StructSchema"></a>
+
+Struct Schema.
+
+#### Initializers <a name="Initializers" id="@winglang/sdk.std.StructSchema.Initializer"></a>
+
+```wing
+new StructSchema(schema: any);
+```
+
+| **Name** | **Type** | **Description** |
+| --- | --- | --- |
+| <code><a href="#@winglang/sdk.std.StructSchema.Initializer.parameter.schema">schema</a></code> | <code>any</code> | *No description.* |
+
+---
+
+##### `schema`<sup>Required</sup> <a name="schema" id="@winglang/sdk.std.StructSchema.Initializer.parameter.schema"></a>
+
+- *Type:* any
+
+---
+
+#### Methods <a name="Methods" id="Methods"></a>
+
+| **Name** | **Description** |
+| --- | --- |
+| <code><a href="#@winglang/sdk.std.StructSchema.asStr">asStr</a></code> | Retrieve the json schema as a string. |
+| <code><a href="#@winglang/sdk.std.StructSchema.validate">validate</a></code> | Attempt to validate a json object against the schema. |
+
+---
+
+##### `asStr` <a name="asStr" id="@winglang/sdk.std.StructSchema.asStr"></a>
+
+```wing
+asStr(): str
+```
+
+Retrieve the json schema as a string.
```

Comment:
maybe additionally instead?

My reason for adding `asStr` was so that if I have a schema
```js
struct Foo {
	val: num;
}
let schema = Foo.schema();
```

Then I can easily just log the schema 

```js
log(schema.asStr())
```

Reply:
Gotcha - yeah, maybe someday we'll support passing `Json` directly to `log`